content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -2,4 +2,940 @@
+detects changes to that file or in any files loaded into the spec file with
+`import` or `require`.
+
+</Alert>
+
+In this case, because we've written a failing test, we should see a red X next
+to the test name, along with a failing assertion of
+`expected false to equal true` in the test body, and a few options for getting
+more information about the failing test.
+
+(SCREENSHOT OF FAILING HELLO WORLD TEST)
+
+### Updating a failing test
+
+Normally, we write assertions that we expect to pass, and refactor our code
+whenever we see an error. However, in this case, the test itself needs to be
+refactored in order for it to pass. Change your existing "should work" test to
+this, and save the spec file:
+
+```js
+it('should work', () => {
+  expect(true).to.equal(true)
+})
+```
+
+Now, the test should pass, and should have a green check mark next to the test
+name, along with a passing assertion of `expected true to equal true` in the
+test body.
+
+(SCREENSHOT OF PASSING HELLO WORLD TEST)
+
+Congratulations! You've written your first test using Cypress component testing.
+Now let's write some meaningful component tests.
+
+## Testing components
+
+- explain general component testing approach
+
+### Setting up imports
+
+In order to test components, we're going to need to import two things into our
+spec file: a function to mount our component, and the component itself.
+
+Different frameworks render their components differently, so we provide
+framework-specific `mount()` functions, which can be imported like so:
+
+<code-group>
+<code-block label="React" active>
+
+```js
+import { mount } from '@cypress/react'
+```
+
+</code-block>
+<code-block label="Vue">
+
+```js
+import { mount } from '@cypress/vue'
+```
+
+</code-block>
+</code-group>
+
+Because our example `LoginForm` component is exported as a named export, we will
+import it into our spec file like so (if the component was exported as a default
+export, we would omit the `{}` curly braces):
+
+```js
+import { LoginForm } from './LoginForm'
+```
+
+### Mounting the component
+
+Now that both the `mount()` function and the component have been imported, we
+can write our first component test.
+
+Replace the contents of your spec file with this, and then save it:
+
+<code-group>
+<code-block label="React" active>
+
+```js
+import { mount } from '@cypress/react'
+import { LoginForm } from './LoginForm'
+
+it('should mount the component', () => {
+  mount(<LoginForm />)
+})
+```
+
+</code-block>
+<code-block label="Vue">
+
+```js
+import { mount } from '@cypress/vue'
+import { LoginForm } from './LoginForm'
+
+it('should mount the component', () => {
+  mount(<LoginForm />)
+})
+```
+
+</code-block>
+</code-group>
+
+Just like in the previous section, we should see one passing test. However, this
+time, because we're mounting a component, we should also see the component
+rendering in the Cypress app.
+
+(SCREENSHOT OF CYPRESS APP WITH MOUNTED COMPONENT)
+
+(link to mount docs ?)
```

Comment:
I'm actually going to just link the React and View `mount()` examples at the bottom in a "See also" section, since there isn't a single documentation page for `mount()`.